src/app/core/routing/store/routing.selectors.ts

```diff
@@ -65,3 +66,15 @@ export const getPrevState = createSelector(getRouterState, prevRoute);
 export const getPrevUrl = createSelector(getRouterState, prevUrl);
 export const getQueryParams = createSelector(getRouterState, queryParams);
 export const getUrl = createSelector(getRouterState, url);
+
+
+export const {
+  selectCurrentRoute, // select the current route
+  selectFragment, // select the current route fragment
+  selectQueryParams, // select the current route query params
+  selectQueryParam, // factory function to select a query param
+  selectRouteParams, // select the current route params
+  selectRouteParam, // factory function to select a route param
+  selectRouteData, // select the current route data
+  selectUrl, // select the current url
+} = getSelectors();
```

Comment:
Opted not to clean up the existing router and store handling but I think opting for the ngrx implementation will ensure a more integrated, tested experience